tests/std/tests/P0009R18_mdspan/test.cpp

```diff
@@ -178,8 +173,8 @@ void extent_tests_copy_ctor_other() {
     static_assert(!is_constructible_v<extents<size_t, 2, 3>, extents<size_t, 3, 2>>);
 
     // Static extents are constructible, but not convertible, from dynamic extents.
-    // static_assert(is_constructible_v<extents<size_t, 2, 3>, extents<size_t, 2, dynamic_extent>>);
-    constexpr extents<size_t, 2, 3> ex0{extents<size_t, 2, dynamic_extent>{}};
+    static_assert(is_constructible_v<extents<size_t, 2, 3>, extents<size_t, 2, dynamic_extent>>);
+    constexpr extents<size_t, 2, 3> ex0{extents<size_t, 2, dynamic_extent>{3}};
     (void) ex0;

```

Comment:
No change requested in this PR (followup OK): Should this test consistently use `[[maybe_unused]]` now instead of `(void)`?

Reply:
> Should this test consistently use `[[maybe_unused]]` now instead of `(void)`?

Yes, but I'm going to remove this test file anyway (after I create separate tests for all classes from this paper).